course7/homeworks/fred/complex-rpn/utils.h

```diff
@@ -0,0 +1,66 @@
+#include "complex.h"
+#include "calculator.h"
+
+#include <string>
+#include <stdexcept>
+#include <utility>
+
+Complex parseComplexFromString(const std::string& s) {
```

Comment:
Don't you thing this logic should be part of the ```Complex``` class?
This is what a user would expect from a complex number library.